CommonTools/Utils/src/FormulaEvaluator.cc

```diff
@@ -657,7 +664,12 @@ namespace {
       return info;
     }
 
-    return info;
+    // throw exception if info still undefined and first char doesn't match variable name
+    if( *iBegin == 'x' or *iBegin == 'y' or *iBegin == 'z' or *iBegin == 't' ) {
```

Comment:
Would be better to reuse the corresponding function from VariableFinder, but wasn't sure how to do. As checkStart of FunctionFinder checks for std::isalpha(iSymbol), all expressions that start with x/y/z/t are also parsed here.

Reply:
@Dr15Jones : I agree with @kirschen that it would be preferable to use here the function coded in VariableFinder::checkStart(). Do you have any suggestion about how to do it?